axum/Cargo.toml

```diff
@@ -42,36 +42,36 @@ __private_docs = ["tower/full", "dep:tower-http"]
 
 [dependencies]
 axum-core = { path = "../axum-core", version = "0.5.0-alpha.1" }
+
+# optional dependencies
```

Comment:
Please remove the changes to this section. It was formatted like this intentionally and the comment especially doesn't make sense now.

Reply:
Ups I ran cargo sort because the CI failed on cargo sort did not notice it changed all of these.